### PR TITLE
Translation update (nl)

### PR DESCRIPTION
**Pull request by dokuwiki-translate**
This pull request contains some translation updates.

### CODE REVIEW

File: lib/plugins/authplain/lang/nl/lang.php [hunk 1]
```diff
@@ -2,9 +2,11 @@
...
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
- * 
+ *
  * @author Hugo Smet <hugo.smet@scarlet.be>
+ * @author Sjoerd <sjoerd@sjomar.eu>
  */
 $lang['userexists']            = 'Er bestaat al een gebruiker met deze loginnaam.';
 $lang['usernotexists']         = 'Sorry, deze gebruiker bestaat niet.';
 $lang['writefail']             = 'Onmogelijk om de gebruikers data te wijzigen. Gelieve de Wiki-Admin te informeren.';
+$lang['protected']             = 'Gegevens van gebruiker %s is beschermd en kan niet worden veranderd of verwijderd';
```
Comment (mprins) [inline lib/plugins/authplain/lang/nl/lang.php]: "Gegevens" is plural so this should be: `Gegevens van gebruiker %s zijn beschermd en kunnen niet worden veranderd of verwijderd`

**Comment by mprins** [comment]
there seems to be a lot of whitespace change / reformatting in this PR for no reason, plus the error I noted.

**Comment by splitbrain** [comment]
whitespace changes are okay (they come from restructuring the English language)

**Comment by Klap-in** [comment]
I will merge it and fix it.

**Comment by Klap-in** [comment]
Done by f2cf01a70b284247ddc42d8f4414bc08e325418a

**Comment by Klap-in** [comment]
Thanks for the notice.

**Pull request merged**